From ce976f49ed507717a2181ddd4946e5ba2e3ce755 Mon Sep 17 00:00:00 2001
From: Vaibhav Bedia <vaibhav.bedia@ti.com>
Date: Thu, 13 Oct 2011 22:24:16 +0530
Subject: [PATCH 0600/1092] arm:debug: Add a config option for enabling JTAG
 clock

In a power aware system all unused clocks should be disabled.
However we might want to enable the JTAG clock alone for debugger
connectivity if such an option exists.

Signed-off-by: Vaibhav Bedia <vaibhav.bedia@ti.com>
Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 arch/arm/Kconfig.debug | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/Kconfig.debug b/arch/arm/Kconfig.debug
index c5213e7..0a9ac1b 100644
--- a/arch/arm/Kconfig.debug
+++ b/arch/arm/Kconfig.debug
@@ -271,4 +271,10 @@ config ARM_KPROBES_TEST
 	help
 	  Perform tests of kprobes API and instruction set simulation.
 
+config DEBUG_JTAG_ENABLE
+	bool "Enable JTAG clock for debugger connectivity"
+	help
+	 Say Y here if you want to enable the JTAG clock to enable
+	 connectivity to a debugger
+
 endmenu
-- 
1.7.11.2

